### PR TITLE
Improve documentation week17

### PR DESCRIPTION
**Pull request by pyhys**

### CODE REVIEW

File: doc/compliancetest.rst [hunk 1]
```diff
@@ -732,21 +745,30 @@ tester software, as we later read the diagnostic log of the PLC.
...
 
 In the TIA portal, make sure you are "Offline" with the PLC (otherwise there
 will be even more additional network load).
 
 Make sure that the software version you run on the DUT has the correct settings,
 for example log level.
 
-* Start the PLC program, and verify that the parameter readout is running (using Wireshark)
+* Verify that the PLC clock setting is correct.
+* Set the device name to ``dut`` with temporaty setting
```
Comment (olbjo) [inline doc/compliancetest.rst:769]: "temporary"

File: doc/creating_gsdml_files.rst [hunk 2]
```diff
@@ -292,13 +294,19 @@ has these attributes:
 * ``SupportedProtocols="SNMP;LLDP"`` Conformance class B must support SNMP.
 * ``PTP_BoundarySupported="true"``
 * ``DCP_BoundarySupported="true"``
-* ``DCP_HelloSupported="true"`` Often used for fast startup.
+* ``DCP_HelloSupported="true"`` Used for fast startup. Then the
+  ``PowerOnToCommReady`` attribute must be set as well.
```
Comment (olbjo) [inline doc/creating_gsdml_files.rst:298]: Remove "Then the". Instead on single line:
``PowerOnToCommReady`` attribute must be set  if DCP_HelloSupported is set to true

**Review by olbjo — APPROVED** [review]
Great work !
Minor typo and a suggestion on gsdml tags description

File: doc/compliancetest.rst (hunk 1 above)
Comment (pyhys) [reply doc/compliancetest.rst]: Done

File: doc/additional_linux_details.rst [hunk 3]
```diff
@@ -55,6 +55,13 @@ build directory by running::
    scan-build cmake --build build
 
 
+Debug compilation issues
```
Comment (hefloryd) [inline doc/additional_linux_details.rst:58]: This is not specific to Linux.

Comment (pyhys) [reply doc/additional_linux_details.rst]: Moved to the end of the Tutorial page.
Done.

**Pull request merged**